templates/new_request_page.hbs

```diff
@@ -14,6 +14,12 @@
     {{t 'submit_a_request'}}
   </h1>
 
+  {{#if parent_id}}
+    <p class="follow-up-hint">
```

Comment:
nit: is this class still needed?

Reply:
Good question!

I didn't find any references to it (in CSS files for example). So in that case that should be okay to remove?